frontend/components/feedback-task/ContentEditableFeedbackTask.vue

```diff
@@ -58,7 +48,7 @@ export default {
     },
     isEditionModeActive: {
       type: Boolean,
-      required: true,
+      default: () => false,
```

Comment:
it's not necessary to put an arrow function as default, you can put directly `default: false`

Reply:
I would rafactor it in other PR because we have many defaults like this.